analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Roslyn/RuleChecks/LocksReleasedAllPathsBase.cs

```diff
@@ -46,6 +47,8 @@ public abstract class LocksReleasedAllPathsBase : SymbolicRuleCheck
             "TryEnterWriteLock"
         };
 
+        protected abstract ISafeSyntaxWalker GetSyntaxWalker(LockAcquireReleaseCollector collector);
```

Comment:
To be explicit about the price of the operation. `Get` should give something that exists.
```suggestion
        protected abstract ISafeSyntaxWalker CreateSyntaxWalker(LockAcquireReleaseCollector collector);
```